_data/projects/fxor.yml

```diff
@@ -0,0 +1,12 @@
+name: fxor
+desc: A tool to encrypt/decrypt a file using XOR operation to do one-time pad.
+site: https://abderraouf-adjal.github.io/fxor
+tags:
+- C
+- Crypto
+- Files
+- Utils
+- Command-line
+upforgrabs:
+  name: fxor
```

Comment:
Instead of using the project name here, this should be the name of the label you use for tagging issues.

We suggest `up for grabs` but you can see from the project list that not everyone uses it...